stream/src/main/scala/org/apache/pekko/stream/impl/PhasedFusingActorMaterializer.scala

```diff
@@ -695,8 +704,6 @@ private final case class SavedIslandData(
     islandName: String,
     subflowFuser: OptionVal[GraphInterpreterShell => ActorRef])
     extends PhaseIsland[GraphStageLogic] {
-  // TODO: remove these
```

Comment:
does 'these' suggests the `logics` should also be removed here?

Reply:
it's been used in `materializeAtomic` method.